src/presentation/test/integration/registration/partnership-type.test.ts

```diff
@@ -0,0 +1,39 @@
+import request from "supertest";
+
+import app from "../app";
+import { WHICH_TYPE_URL } from "../../../controller/registration/Routing";
+import { appDevDependencies } from "../../../../config/dev-dependencies";
+import enTranslationText from "../../../../../locales/en/translations.json";
+import RegistrationPageType from "presentation/controller/registration/PageType";
```

Comment:
the import looks like it is missing the relative path ../../../etc. is it ok?

Reply:
import fixed